public/locales/en/panel-nightsky.json

```
@@ -0,0 +1,249 @@
+        }
+      }
+    },
+    "labels": {
+      "title": "Labels",
+      "input": {
+        "show-labels": "Show Labels",
+        "show-alternate-labels": "Show Alternate Labels"
+      }
+    },
+    "appearance": {
+      "title": "Apperance",
+      "warning": {
+        "title": "Only Night Sky Profile",
+        "description": "The buttons below only work in the 'nightsky' profile. Use the profile editor to edit the actions and adjust the values for your dome."
+      },
+      "buttons": {
+        "default-settings": "Default Settings",
+        "point-like-stars": "More Point-like Stars"
+      }
+    }
+  },
+  "sun": {
+    "glare": {
+      "title": "Glare",
+      "buttons": {
+        "show-glare": "Show Glare",
+        "hide-glare": "Hide Glare"
+      }
+    },
+    "size": {
+      "title": "Size",
+      "buttons": {
+        "default-size": "Default Angular Size",
+        "large-size": "Large Angular Size",
+        "extra-large-size": "Extra Large Angular Size"
+      },
+      "aria-labels": {
+        "increase-size": "Increase angular size",
+        "decrease-size": "Decrease angular size"
+      },
+      "error": "Could not find Sun Angular Size settings"
+    },
+    "trails": {
+      "title": "Trails",
+      "buttons": {
+        "add-trail": "Add Trail",
+        "add-trail-simulation-date": "Add Trail for Simulation Date",
+        "add-trail-today": "Add Trail for Today",
+        "hide-trails": "Hide All"
+      },
+      "choose-date": "Choose Date",
+      "added-trails": {
+        "title": "Added Sun Trails",
+        "no-trails": "No Sun trails"
+      }
+    }
+  },
+  "time": {
+    "simulation-time": {
+      "title": "Simulation Time",
+      "buttons": {
+        "set-now": "Set to Now"
+      },
+      "local": "Local",
+      "timezone": "Timezone",
+      "utc": "UTC",
+      "error": "Date out of range"
+    },
+    "jumps": {
+      "title": "Jumps",
+      "buttons": {
+        "sidereal-day": {
+          "title": "Sidereal Day",
+          "decrease": "Go back in time one sidereal day",
+          "increase": "Go forward in time one sidereal day"
+        },
+        "sidereal-week": {
+          "title": "Sidereal Week",
+          "decrease": "Go back in time one sidereal week",
+          "increase": "Go forward in time one sidereal week"
+        },
+        "solar-day": {
+          "title": "Solar Day",
+          "decrease": "Go back in time one solar day",
+          "increase": "Go forward in time one solar day"
+        },
+        "solar-week": {
+          "title": "Solar Week",
+          "decrease": "Go back in time one solar week",
+          "increase": "Go forward in time one solar week"
+        }
+      }
+    },
+    "diurnal-motion": {
+      "title": "Diurnal Motion",
+      "buttons": {
+        "realtime": "Realtime",
+        "60x": "60x (1 min/sec)",
+        "120x": "120x (2 min/sec)",
+        "300x": "300x (5 min/sec)",
+        "play-pause": "Play / Pause"
```

```suggestion
        "60x": "60x (1 min/s)",
        "120x": "120x (2 min/s)",
        "300x": "300x (5 min/s)",
        "play-pause": "Play / Pause"
```
for consistency with the other places we display time?